tests/core/_orchestrator/test_orchestrator__submit_task.py

```diff
@@ -46,14 +47,15 @@ def test_submit_task_development_mode():
     scenario = create_scenario()
     orchestrator = _OrchestratorFactory._build_orchestrator()
 
-    submit_time = datetime.now()
+    submit_time = datetime.now() + timedelta(seconds=1)  # +1 to ensure the edit time of dn_0 is before the submit time
     with freezegun.freeze_time(submit_time):
         submission = orchestrator.submit_task(
             scenario.t1, no_of_retry=10, log=True, log_file="file_path"
         )  # t1 is executed directly in development mode
         job = submission.jobs[0]
 
     # task output should have been written
+    assert scenario.dn_0.last_edit_date < submit_time
```

Comment:
This assertion does not belong to the test_submit_task function. It should be part of the create scenario test function. No?

Reply:
you're right